docs/resources/elb_loadbalancer.md

```diff
@@ -156,6 +156,12 @@ The following arguments are supported:
 * `auto_renew` - (Optional, String, ForceNew) Specifies whether auto renew is enabled. Valid values are **true** and
   **false**. Changing this parameter will create a new resource.
 
+* `enable` - (Optional, Bool) Specifies whether autoscaling is enabled. Valid values are **true** and
```

Comment:
suggest to rename it to **autoscaling_enabled**

Reply:
done